### DISCUSSION TITLE
docker tag pointing to latest stable release

### DISCUSSION THREAD

**Discussion by totallyben** [nginx-proxy/nginx-proxy#2480] [Ideas]
Following on from https://github.com/nginx-proxy/nginx-proxy/issues/2469#issuecomment-2139301578 it would be really helpful if there was a tag pointing to the latest stable release.

The reason behind this is that we have large number of clients that require security updates to be applied automatically and having to increment version numbers manually is not really feasible.  

We could of course create additional scripts to do this but adding a new tag would really help and negate the need for this.

**Comment by buchdag** [comment]
I've given this some thought and I don't think a `stable` tag would improve anything over `latest`. What happen when a breaking change happen and we go from 1.x to 2.x ? Do we need to create a new `stable-v2` tag  and stop updating `stable` (I don't think we have the ressources to provide security update to previous major versions) ? Even without major version change in mind, the reality is that this project don't have the user feedback, metrics or ressources to absolutely guarantee that a given release is going to be "stable" compared to the previous one, beside me trying to be careful and not breaking existing setup.

I don't think I can do much more that what is already in place: releasing regularly with semantic versioning, documenting changes on release notes and warning when something breaking or unusual happen. If you want automatic updates without reading release notes or using semver tags, a `stable` tag that is a few week behind `latest` won't improve stability in any measurable way and actually negate the avantages of having versioned releases in the first place, in the same way `latest` does (ie you won't be able to tell what version you where running before an update caused issues and conversely you won't be able to immediately roll back to the previous working version).
